### PR TITLE
LIIKUNTA-149 & LIIKUNTA-150 & LIIKUNTA-151 | Add translations

### PR DESCRIPTION
**Pull request by pjlampil**
## Description

Add batch of the translations (especially English and Swedish). A couple of items were not translated (for unknown reason)

### Closes

**[LIIKUNTA-149](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-149)**
**[LIIKUNTA-150](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-150)**
**[LIIKUNTA-151](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-151)**

### CODE REVIEW

File: public/locales/en/date_time_picker.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  "date_input.error.default_max_date": "EN: Valittu päivä on myöhäisimmän sallitun päivän jälkeen"
+  "select": "Select",
+  "close": "Close",
+  "date_input.label": "Day",
+  "date_input.helper_text": "Use the format D.M.YYYY",
+  "time_input.label": "Time",
+  "time_input.hours_label": "Hours",
+  "time_input.minutes_label": "Minutes",
+  "date_input.error.default_min_date": "The selected date is earlier than the earliest allowed date",
+  "som.har.valts.in.är.före.det.tidigast.tillåtna.datumet": "The selected date is later than the latest allowed date"
```
Comment (tuovinensanttu) [inline public/locales/en/date_time_picker.json:10]:
```suggestion
  "date_input.error.default_max_date": "The selected date is later than the latest allowed date"
```
Translation key is in swedish.

File: public/locales/fi/date_time_picker.json [hunk 2]
```diff
@@ -7,5 +7,5 @@
   "time_input.hours_label": "Tunnit",
   "time_input.minutes_label": "Minuutit",
   "date_input.error.default_min_date": "Valittu päivä on ennen aikaisinta sallittua päivää",
-  "date_input.error.default_max_date": "Valittu päivä on myöhäisimmän sallitun päivän jälkeen"
+  "som.har.valts.in.är.före.det.tidigast.tillåtna.datumet": "Valittu päivä on myöhäisimmän sallitun päivän jälkeen"
```
Comment (tuovinensanttu) [inline public/locales/fi/date_time_picker.json:10]:
```suggestion
  "date_input.error.default_max_date": "The selected date is later than the latest allowed date"
```
Translation key is in swedish.

File: public/locales/sv/date_time_picker.json [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-  "date_input.error.default_max_date": "SV: Valittu päivä on myöhäisimmän sallitun päivän jälkeen"
+  "select": "Välj",
+  "close": "Stäng",
+  "date_input.label": "Datum",
+  "date_input.helper_text": "Ange i formen D.M.ÅÅÅÅ",
+  "time_input.label": "Tid",
+  "time_input.hours_label": "Timmar",
+  "time_input.minutes_label": "Minuter",
+  "date_input.error.default_min_date": "Dagen som har valts in är före det tidigast tillåtna datumet",
+  "som.har.valts.in.är.före.det.tidigast.tillåtna.datumet": "Dagen som har valts in är efter det senast tillåtna datumet"
```
Comment (tuovinensanttu) [inline public/locales/sv/date_time_picker.json:10]:
```suggestion
  "date_input.error.default_max_date": "The selected date is later than the latest allowed date"
```
Translation key is in swedish.

File: public/locales/en/date_time_picker.json (hunk 1 above)
Comment (pjlampil) [reply public/locales/en/date_time_picker.json]: Good catch 👀

**Review by tuovinensanttu — APPROVED** [review]
LGTM 👍

**Pull request merged**